### PR TITLE
Add command option to export only a set of chats

### PR DESCRIPTION
**Pull request by rajkumaar23**
This feature will let you export history of only a set of chats rather than all, specified by `-c`.

### CODE REVIEW

**Comment by rajkumaar23** [comment]
@carderne Let me know if you would like to have any changes made to this feature.

File: sigexport.py [hunk 1]
```diff
@@ -410,6 +414,9 @@ def merge_with_old(dest, old):
 @click.option(
     "--source", "-s", type=click.Path(), help="Path to Signal source and database"
 )
+@click.option(
+    "--chat", "-c", type=click.Path(), help="Set of chat names to be exported"
```
Comment (carderne) [inline sigexport.py:418]: Delete the `type=` parameter, as it should be `STRING`, which is anyway the default.

Comment (carderne) [reply sigexport.py:418]: And maybe add what the chat 'names' are? Assume it's the contact name or group name?
e.g. `help="Comma-separated chat names to include. These are contact names or group names`.

Comment (rajkumaar23) [reply sigexport.py]: Sure, I have made those changes.

**Comment by carderne** [comment]
Thanks @rajkumaar23

**Pull request merged**